internal/clone/doc.go

```diff
@@ -0,0 +1 @@
+package clone
```

Comment:
Was there going to be a package doc comment?

Reply:
You mention this in the pr description, nevermind.

Reply:
Since the only files in this repo had windows build tags on them, the opening on linux would complain that it couldn't find the packaged (since all files were excluded).
I added this just to have a placeholder file to satisfy the import.
I think the package docs would be added in a separate PR.